hal/include/HAL/Accelerometer.h

```diff
@@ -9,11 +9,14 @@
 
 #include "HAL/Types.h"
 
-enum HAL_AccelerometerRange : int32_t {
+/* clang-format off */
```

Comment:
The point of this commit in part is to make it C compatible.  Since there are literally 2 places in the code that don't use block comments, at least that are supposed to be C compatible, we should just use the ANSI C block comment system.